tests/function_libs/torch_lib/ops_test_data.py

```diff
@@ -438,6 +438,34 @@ def _where_input_wrangler(
     return args, kwargs
 
 
+def _unique_unsorted_xfail_matcher(
+    sample: Any
+) -> bool:
+    # torch.unique always sorts, so the results are not guaranteed to be
+    # equivalent to the output of the ONNX op
+    expect_fail = False
+    if sample.kwargs.get("sorted", None) is False and sample.input.numel() > 1:
+        # sorted == None is equivalent to True
+        # the result will be mismatched if the input is not sorted
+        # To expect equality, we must verify that the first appearance
+        # of each unique value is in ascending order in the input
+        test_kwargs = dict(sample.kwargs)
+        test_kwargs['return_inverse'] = True
+        test_kwargs['return_counts'] = False
+        _, inverse = torch.unique(sample.input, **test_kwargs)
+        observed = set()
+        max_observed: Optional[int] = None
+        for inv_idx in inverse.flatten().tolist():
+            if inv_idx not in observed:
+                if max_observed is not None and inv_idx < max_observed:
+                    expect_fail = True
+                    break
+                observed.add(inv_idx)
+                if max_observed is None or inv_idx > max_observed:
+                    max_observed = inv_idx
+    return expect_fail
```

Comment:
```suggestion
    return False
```

Possible to simplify?